script.js

```diff
@@ -15,13 +17,224 @@ navPort.addEventListener('click', () => {
+  navbar.classList.toggle('active');
+  fixed.classList.toggle('fixed-active');
+});
+
+navContact.addEventListener('click', () => {
+  navbar.classList.toggle('active');
+  fixed.classList.toggle('fixed-active');
+});
+
+const CardArray = [{
+  title: 'Multi-Post Stories Gain+Glory',
+  languageTags: ['Ruby on rails', 'Css', 'Javascript', 'Html'],
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release',
+  modalCardImage: './images/detail-snapshot.png',
+},
+
+{
+  title: ' Keeping track of components',
+  languageTags: ['Ruby on rails', 'Css', 'Javascript', 'Html'],
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release',
+  modalCardImage: './images/modal-desk-pic.png',
+},
+
+{
+  title: 'Multi-Post Stories Gain+Glory',
+  languageTags: ['Ruby on rails', 'Css', 'Javascript', 'Html'],
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release',
+  modalCardImage: './images/detail-snapshot.png',
+},
+
+{
+  title: ' Keeping track of components',
+  languageTags: ['Ruby on rails', 'Css', 'Javascript', 'Html'],
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release',
+  modalCardImage: './images/modal-desk-pic.png',
+},
+
+{
+  title: 'Multi-Post Stories Gain+Glory',
+  languageTags: ['Ruby on rails', 'Css', 'Javascript', 'Html'],
```

Comment:
- Please on your mobile view, the popup elements are not properly aligned as in the Figma design. Please try to fix the below issues.
  - Please on your design the project description is text-centered, while that of the Figma is not centered, please work with the Figma design. so that your design looks closes like that of the Figma design
  - Please take a look at the `See live and See source` The spaces between the text and the icon are small compared to that In the Figma design, please space them appropriately.  

##### Your design
![Screenshot from 2022-06-16 18-44-06](https://user-images.githubusercontent.com/58771507/174133991-96d8fe8e-1660-423b-aa58-9a0a80668b9d.png)

##### Figma design

![Screenshot from 2022-06-16 18-47-43](https://user-images.githubusercontent.com/58771507/174134093-2899ad49-59da-4bdf-8b6a-19c8873a5aec.png)

Reply:
Thank you for calling our attention to the error. We will resolve it.